src/ucp/wireup/select.c

```diff
@@ -129,7 +129,8 @@ static int ucp_wireup_is_reachable(ucp_worker_h worker, ucp_rsc_index_t rsc_inde
 static UCS_F_NOINLINE ucs_status_t
 ucp_wireup_select_transport(ucp_ep_h ep, const ucp_address_entry_t *address_list,
                             unsigned address_count, const ucp_wireup_criteria_t *criteria,
-                            uint64_t tl_bitmap, uint64_t remote_md_map, int show_error,
+                            uint64_t tl_bitmap, uint64_t remote_md_map,
+                            uint64_t remote_addr_bitmap, int show_error,
```

Comment:
wouldn't be simpler to pass remote_dev_bitmap and local_dev_bitmap?
so will not need XX_unset_by_device

Reply:
it will require more global changes in transport selection routine, ok, will do